innovation_sweet_spots/analysis/wrangling_utils.py

```diff
@@ -64,7 +64,9 @@ def get_project_funds(self, gtr_projects: pd.DataFrame) -> pd.DataFrame:
             gtr_projects.merge(self.link_gtr_funds, on="project_id", how="left")
             # Add funding amounts and dates to the table
             .merge(self.gtr_funds, on="id", how="left")
-            .query("category=='INCOME_ACTUAL'")
+            .query(
+                "category==['INCOME_ACTUAL', 'EXPENDITURE_ACTUAL', 'EXPENDITURE_PROJECTED']"
+            )
```

Comment:
I've added `EXPENDITURE_ACTUAL` and `EXPENDITURE_PROJECTED` here to the query as otherwise a few records for `fund_start` and `fund_end` would have `NaT` values and cause issues when splitting the data.
There are 3 projects with `category == EXPENDITURE_ACTUAL` and 17 projects with `category == EXPENDITURE_PROJECTED`
Do we want to have these records in the dataset?

Reply:
The project ids with `EXPENDITURE_ACTUAL` are:
['4DE6E882-AE1B-4EFE-BDD4-9C3BBBADF89C',
'7D4E282A-7925-4C4B-A7DE-E3552D6172D4',
'33D5D968-04A9-462F-8F68-4E3BF4A4255B']

The project ids with `EXPENDITURE_PROJECTED` are:
['B07E5E3B-2EF9-4EC6-B4DD-E54DCC7F18A0',
'77B0CFF0-A57E-4FE9-9E8B-B20B30F133CD',
'6B6ACAF3-80A6-479E-A325-DD6E247F1014',
'4FB6E64E-C62B-4B08-A4E9-E02B9A931127',
'4BD19E74-7969-4E3A-9AB3-4956DB381B07',
'258D1B6B-0F56-4943-B4A1-48D8B9D1DD2E',
'0B5FF730-E385-4264-940F-CE19C86C1404',
'05158845-C9A5-488A-977D-AD0B21B07D01',
'C8865CED-ED83-4E68-8BEA-07E06E2DB150',
'5D64B9E2-D3DF-4813-A22A-1E6ED3ED2636',
'A8FFFF2D-0AC3-48A8-BB01-D1882E434BA2',
'13195374-764E-4933-8F2E-5646DF09CA1F',
'868E4A96-E55A-4F7A-8A48-ED6045CB1E30',
'51DFFCA6-9824-49B2-A564-53550B9F12C3',
'4BA654E2-BEE2-4081-8CEB-D2CA00D1EF36',
'7665CD4A-C1CB-460B-93C1-AF6E129E23E5',
'B8004F55-0011-4F40-B1D5-BF3E9E0C8252']

Reply:
If you take a look at these projects, is there anything special about them? (eg, perhaps they haven't finished yet...)

Reply:
[Here's a sheet](https://docs.google.com/spreadsheets/d/1Wf67ZELQ94-oJuDvtkM6dC2S-INDofrwkPmh2XqAxh4/edit#gid=0) with all the non `INCOME_ACTUAL` records. 
I have added a column with links to the UKRI GtR website. The records seem correct. Most of the records are `Intramural`.

Reply:
I think your solution to include the other categories as well is perfectly fine - thank you for spotting and adding this!

FYI I checked the other 'Intramural' grants across all GtR data, and it does look like, as you've noted, that the vast majority of those are coding funding as `EXPENDITURE_...` (whereas `Research Grants` for example are all `INCOME_ACTUAL`).

Reply:
Thanks for checking 🙂